saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/ClassifierUtils.scala

```diff
@@ -19,48 +19,48 @@ object ClassifierUtils extends Logging {
     def apply[T <: AnyRef](c: (Learnable[T], Iterable[T])*) = {
       c.foreach {
         case (learner, trainInstances) =>
-          logger.info(evalSeparator)
-          logger.info("Training " + learner.getClassSimpleNameForClassifier)
+          println(evalSeparator)
```

Comment:
Is the change from `logger.info` to `println` intentional? No strong opinion, just wanted to confirm.